src/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
- [ ] It would be better if we add instructions about how to clone and run the app locally in our README file so that hiring managers with no coding or non-technical skills can also appreciate and run the app. For example; `git clone https://github.com/leolpaz/To-Do-list.git`, install dependencies with `npm i`, start the app with `npm start`